### DISCUSSION TITLE
Cannot create Database without credit or debit card

### DISCUSSION THREAD

**Discussion by INDERJEET0202** [planetscale/discussion#600] [Q&A]
I am a student and I don't have any credit card, I have only a VISA debit card which is not supported in planetscale. Please suggest me what can I do? @gedgar @johnyerhot @vmg

**Comment by orware** [comment]
Hello @INDERJEET0202,

I also see that you have posted over in https://github.com/planetscale/discussion/discussions/510#discussioncomment-8237269 as well.

The likely explanation for why your debit card was not accepted would be due to it being a prepaid card and/or the required e-mandate phase of the process not being completed, which mainly provides a pre-approval for any possible future usage on a paid plan.

For future support needs/issues please reach out to us directly over at https://support.planetscale.com/ so we can assist you best!
